### PR TITLE
Update dependencies to new revision

### PR DESCRIPTION
**Pull request by azoyan**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,23 +1,38 @@
 [package]
 name = "fungible-token"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[workspace.package]
 version = "0.1.5"
```
Comment (bezrazli4n0) [inline Cargo.toml]: Please, change version to 0.1.6

**Review by bezrazli4n0 — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -5,8 +5,3 @@ updates:
     labels: ["A4-insubstantial"]
     schedule:
       interval: "daily"
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    labels: ["A4-insubstantial"]
-    schedule:
-      interval: "daily"
```
Comment (fluiderson) [inline .github/dependabot.yml]: Why is this removed?

**Review by fluiderson — APPROVED** [review]
